### ISSUE TITLE
is requiring Radix really necessary?

### ISSUE THREAD

**Issue by fiatjaf** [feross/standard#384]
https://github.com/eslint/eslint/blob/master/docs/rules/radix.md

I don't see why would Standard bother to require this, since ECMAScript itself has changed to actually stop requiring developers to write the radix parameter.

**Comment by rstacruz** [comment]
It's not required, but it can be a problem. Consider:

```js
input = document.getElementById('textbox')
parseInt(input.value)
```

If I wrote `030`, it'd behave differently in older browsers. This quirk has since been removed in newer engines, though, so I'm not sure on what the right answer should be today.

(For what it's worth, I personally prefer to use `+input.value` to numberify a string)

**Comment by fiatjaf** [comment]
That's explained in that link. I only think `parseInt` is written without the radix by the majority of people, and it is not a big issue anywhere, so this Standard shouldn't be forcing people to use the radix. They probably know what they are doing, right? Not always, but it doesn't matter.

Anyway, it's just a minor quibble. I can start writing the radix from now on.

**Comment by yoshuawuyts** [comment]
From [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Octal_interpretations_with_no_radix):
> Many implementations have not adopted this behavior as of 2013, and because older browsers must be supported, always specify a radix.

+0 from me, I don't feel strongly about this either way.

**Comment by feross** [comment]
Looks like things have changed since 2013. Just tested latest Chrome, Firefox, Safari, and Edge and `parseInt('071')` returns `71` as expected.

I think we can finally relax this rule. Code that uses `parseInt` and targets really old browsers should still use a radix, but this doesn't need to be a rule anymore.

This will be released in standard v6.0.0.
